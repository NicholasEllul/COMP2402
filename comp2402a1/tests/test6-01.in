Hello
My
name
is
nick

heres a blank
line
lint
my
my
my m
