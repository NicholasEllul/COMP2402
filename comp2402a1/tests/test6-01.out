
My
is
my
line
lint
my m
name
nick
Hello
heres a blank
